apps/web/components/Charts/v2/renderers/ChartRenderer.tsx

```diff
@@ -39,6 +39,64 @@ type ChartProps = {
   slice: IChart
 }
 
+const transformData = (items: any) => {
+  return {
+    statisticsForHeader: items
+      .flatMap((item: { categoryItems: any }) => item.categoryItems)
```

Comment:
Are "categoryItems" always something that is used when transforming data?

Reply:
Yes, transformData funtion is just for pie charts, i think its best for now just use category items for pie charts